### PR TITLE
Add rack leg scene and refine moveit scene operation

### PR DESCRIPTION
**Pull request by knorth55**
cardboard_rack_scene

- add rack leg scene to avoid collision.
-  refine moveit scene operation in main
   - now it takes 0.5 seconds, so I remove unnecessary moveit scene operations.
- calibrate `cardboard_marker.yaml`
- refine `:place-object` motion

### CODE REVIEW

**Comment by knorth55** [comment]
I tested with real robot.

File: jsk_arc2017_baxter/euslisp/lib/pick-interface.l [hunk 1]
```diff
@@ -250,21 +248,40 @@
...
+        (if (eq target-cardboard :c)
+          (progn
+            (pushback
+              (if (eq grasp-style :suction)
+                (send self :ik->bin-center arm :b :offset #f(0 0 400)
+                      :rotation-axis :z :use-gripper t)
+                (send self :ik->bin-center arm :b :offset #f(0 0 400)
+                      :rotation-axis nil :move-palm-end t))
+              avs)
+            (setq approach-z 200)
```
Comment (pazeshun) [inline jsk_arc2017_baxter/euslisp/lib/pick-interface.l]: Where do you use approach-z?

Comment (knorth55) [inline jsk_arc2017_baxter/euslisp/lib/pick-interface.l]: it is typo of `place-z`

File: jsk_arc2017_baxter/euslisp/lib/pick-interface.l [hunk 2]
```diff
@@ -250,21 +248,40 @@
...
+              avs)
+            (setq place-z 200)
+            (pushback
+              (if (eq grasp-style :suction)
+                (send self :ik->cardboard-center arm target-cardboard
+                      :offset (float-vector 0 0 100) :rotation-axis :z :use-gripper t)
+                (send self :ik->cardboard-center arm target-cardboard
+                      :offset (float-vector 0 0 100) :rotation-axis nil :move-palm-end t))
+              avs)
+            (setq place-z 50))
```
Comment (pazeshun) [inline jsk_arc2017_baxter/euslisp/lib/pick-interface.l]: You set place-z twice. 
Which is correct?

Comment (knorth55) [inline jsk_arc2017_baxter/euslisp/lib/pick-interface.l]: updated

**Comment by wkentaro** [comment]
Please resolve the conflict.

**Pull request merged**